dataprofiler/profilers/profile_builder.py

```diff
@@ -322,12 +368,25 @@ def clean_data_and_get_base_stats(df_series, sample_size,
         if not len_df:
             return df_series, {
                 "sample_size": 0, "null_count": 0,
-                "null_types": dict(), "sample": []
+                "null_types": dict(), "sample": [],
+                "min_id": None, "max_id": None
             }
 
         # Pandas reads empty values in the csv files as nan
         df_series = df_series.apply(str)
 
+        # Record min and max index values if index is int
+        min_id = None
+        max_id = None
+        if isinstance(df_series.index, pd.RangeIndex) or \
+                all([isinstance(i, int) for i in df_series.index]):
+            min_id = min(df_series.index)
+            max_id = max(df_series.index)
+        else:
+            warnings.warn("Unable to detect minimum and maximum index values. "
+                          "As a result overlap when merging/updating cannot be "
+                          "detected and may result in an inaccurate profile.")
```

Comment:
I think this can be refined such that the user understands what's going on. I don't think the phrase: "As a result overlap when merging/updating cannot be detected and may result in an inaccurate profile." is disconcerting and I'm not sure relays how they user can fix it and / or what they should be concerned about.